sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContainerType.md

```diff
@@ -0,0 +1,69 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/remove-spocontainertype
+applicable: SharePoint Online
+title: Remove-SPOContainerType
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: ShreyasSar26
+ms.reviewer:
+---
+ 
+# Remove-SPOContainerType
+ 
+## SYNOPSIS
+This cmdlet removes the container type specified from the tenant.
+ 
+## SYNTAX
+ 
+### ParamSet1
+ 
+```powershell
+Remove-SPOContainerType -ContainerTypeId <ContainerTypeId>
+```
+ 
+## DESCRIPTION
+ 
+This cmdlet deletes only the trial container type in the tenant. To delete a container type in trial status, you must remove all containers of the container type first, including from the deleted container collection. To remove containers, refer to Consuming Tenant Admin. Once all the containers are deleted, trial containertype can be deleted using the PowerShell cmdlet.
```

Comment:
>Consuming Tenant Admin [](http://example.com/codeflow?start=249&length=22)

what is this? should it be a link to a documentation?

Reply:
Yes. Adding the link to the documentation